src/holo-test

```diff
@@ -39,9 +39,9 @@ run_testcase() {
     # setup chroot for holo run
     rm -rf -- target/
     cp -R source/ target/
+    mkdir -p target/run
+    mkdir -p target/tmp
     mkdir -p target/usr/share/holo/files
-    mkdir -p target/usr/share/holo/run-scripts
-    mkdir -p target/usr/share/holo/users-groups
```

Comment:
Oh yeah, I forgot about `holorc.d` (although I don't know of any testcases using that). Re the regex: I actually checked `holo-plugin-interface(7)` for acceptable chars in a plugin ID, in order not to be bitten by possible future extensions of `holorc` syntax.